clients/python/pyproject.toml

```
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "model-registry"
+version = "0.1.0"
+description = "Client for Red hat OpenDataHub Model Registry"
```

```suggestion
description = "Client for OpenDataHub Model Registry"
```

Very minor, but I think being the community version, they would prefer not to prefix it, to inadvertently give wrong impression?
If you want to keep it, it's fine too, but capital H per brand guidelines :) `Red Hat`

---

Oh MB :) thx for the suggestion.